### PR TITLE
Fix navbar menu and profile

### LINKED ISSUE #39: Remove Weclome, display the name of the logged in user in the top bar
Could be placed temporarily next to the logo ?

### PR DESCRIPTION
**Pull request by seegwen**
Is based on PR #47

Will partially address issue #44 (not the map button, since we have nothing to show there yet)
fixes #39

### CODE REVIEW

**Comment by vaab** [comment]
Can you rebase this on main ? and check it. I don't see the user name. I'll review it once rebased on main.

File: src/components/Nav.vue [hunk 1]
```diff
@@ -33,22 +33,35 @@
...
           <div class="buttons">
-            <router-link v-if="!getLog" to="/" class="button is-light">
+            <router-link to="/" class="button is-light">
               Se connecter
             </router-link>
-            <router-link v-if="getLog" to="/profile" class="button is-light">
-              Profile
-            </router-link>
-            <router-link v-if="getLog && hasUnconfiguredBackend" to="/create-account" class="button is-light">
+          </div>
```
Comment (vaab) [inline src/components/Nav.vue:41]: You can solve #57 if you want (optional !, this can be in another PR). Yes I just added it !

File: src/components/Nav.vue [hunk 2]
```diff
@@ -33,22 +33,35 @@
...
+        </div>
+
+        <div class="navbar-item has-dropdown is-hoverable" v-if="getLog">
```
Comment (vaab) [inline src/components/Nav.vue:44]: I always have a dropdown now, is that intended ? I'm not sure it was requested.

File: src/components/Nav.vue [hunk 3]
```diff
@@ -33,22 +33,35 @@
...
+          <a class="navbar-link">
+            {{ userProfile ? userProfile.name : 'inconnu' }}
+          </a>
```
Comment (vaab) [inline src/components/Nav.vue:47]: For some reason, I don't get anything diplayed in the bar. I didn't dig further to be honest. I hope it is not coming from me, it is still possible. Anyway, your rebase is welcome and quick double check is welcome on this point.

File: src/components/Core.vue [hunk 4]
```diff
@@ -1,15 +1,10 @@
 <template>
   <main class="main">
     <div class="container mt-5">
-      <section class="top-column column">
-        <h1 class="title is-size-4 has-text-grey has-text-weight-medium mb-4">Bienvenue {{userProfile ? userProfile.name : 'inconnu'}}</h1>
-      </section>
-      <!-- conteneur des colonnes de gauche et droite où se situent les cards -->
```
Comment (vaab) [inline src/components/Core.vue:7]: Nice, there still seem to have a lot more code you can remove in this commit and same very file. Am I wrong ? (computed properties and some part of the `mounted` callback.

**Review by vaab — CHANGES_REQUESTED** [review]

File: src/components/Nav.vue (hunk 2 above)
Comment (seegwen) [reply src/components/Nav.vue:44]: Intended, since we're about to add the map section, and we now have a proper profile page (on the Odoo instance), it seemed to me it would be worth it moving "Profile" and "Disconnect" into the user dropdown. We talked about it in another issue or PR IIRC.

File: src/components/Nav.vue (hunk 3 above)
Comment (seegwen) [reply src/components/Nav.vue:47]: I see. It's because it depended on the big PR #53 for the userprofile fetching in the store.

File: src/components/Core.vue (hunk 4 above)
Comment (seegwen) [reply src/components/Core.vue:7]: You're right, but again, I think this is addressed in another PR and I was afraid to mix things up. I'll try and clean the useless code in this one as you requested.

**Comment by seegwen** [comment]
All issues addressed, and this PR also fixes #57

File: src/components/Nav.vue (hunk 2 above)
Comment (seegwen) [reply src/components/Nav.vue:37]: It was not requested and you rightfully asked so I asked back to mathieu and stephan and they agree on the change.

File: src/components/Nav.vue (hunk 1 above)
Comment (seegwen) [reply src/components/Nav.vue]: Resolved

File: src/components/Core.vue (hunk 4 above)
Comment (seegwen) [reply src/components/Core.vue:7]: Cleaned

**Comment by seegwen** [comment]
Now there is a new getter to pull the odooUrl from the lokapi.
partial fix for #44

**Comment by seegwen** [comment]
@vaab reverted from the dropdown menu.

**Comment by vaab** [comment]
Thanks a lot, they are integrated and ready to push.
